scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
   // below the smallest value in the build output; this avoids the
   // need for frequent changes as bundle sizes fluctuate.
   MIN_SIZE: 574,
-  MAX_SIZE: 877,
+  MAX_SIZE: 884,
```

Comment:
What has caused this big jump? Are we getting duplicate sub-dependencies because of how `yarn` resolves packages? May be worth a look at some of the trees for psammead packages for example